modules/stanford_paragraph_options/stanford_paragraph_options.module

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @file
+ * Contains stanford_paragraph_options.module.
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+use \Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_help().
+ */
+function stanford_paragraph_options_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    // Main module help for the stanford_paragraph_options module.
+    case 'help.page.stanford_paragraph_options':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('Limit the options of paragraph types for fields.') . '</p>';
+      return $output;
+
+    default:
+  }
+}
+
+/**
+ * Implements hook_form_alter().
+ */
+function stanford_paragraph_options_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
I think we should be more specific on the form we are altering. Right now this is an "every form" hook. Can you narrow it down to the complex page's field? I think that is really the only place we want to do this as we are trying to avoid setting it in the feature.